### ISSUE TITLE
Recover outfits from locked storage

### ISSUE THREAD

**Issue by TomGoodIdea** [endless-sky/endless-sky#8249]
### Problem Description

@daggs1 suggested to give the player a chance to recover their planetary outfit storage if the planet no longer has an outfitter. I have a few implementation ideas, and decided to make it a separate issue.

### Related Issue Links

Original suggestion:
https://github.com/endless-sky/endless-sky/pull/8215#issuecomment-1406801803

### Desired Solution

Add a "Storage" button to all planets with a spaceport, but without an outfitter. The storage panel would be similar to the outfitter panel, but the player couldn't install or uninstall outfits, only move them between the storage and ship's cargo.

However, this would allow storing outfits on **all** planets that have a spaceport (not sure if we want this).

### Alternative Approaches

The same as above, but make the storage panel displaying only on planets with stored outfits, and not accepting any new outfits if there's no outfitter (the player could only take things out)

**Comment by daggs1** [comment]
> 

not necessarily, if the is a regime change (like the game has in some storylines), then the outfits are merely inaccessible until you have the chance to land and access the storage location
